src/PoolManager.sol

```diff
@@ -314,8 +325,10 @@ contract PoolManager is Auth {
         require(liquidityPool == address(0), "PoolManager/liquidityPool-already-deployed");
         require(pools[poolId].createdAt != 0, "PoolManager/pool-does-not-exist");
 
+        address[] memory liquidityPoolWards = new address[](1);
+        liquidityPoolWards[0] = address(investmentManager);
```

Comment:
why passing the wards to the factory, if we pass the investment manager address anyway.

Reply:
So that in the future if we might need another ward, we don't need to modify the factory contract, which would be a bytecode change and thus change the address.